packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -153,7 +154,20 @@ class CloudFormation {
   }
 
   readStackEvents(stackName) {
-    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME);
+    const self = this;
```

Comment:
Do we need this rename?

Reply:
yup.
<img width="629" alt="image" src="https://user-images.githubusercontent.com/5849952/208465771-6aee1bc5-2a8d-4534-840d-504f82d48705.png">